models/public/ssd-resnet34-1200-onnx/model.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+    The "ssd-resnet-34-1200-onnx" model is a multiscale SSD based on ResNet-34
```

Comment:
I'm not convinced this should have an `-onnx` suffix. The framework that the model was actually created with is PyTorch, right? So IMO, it would make more sense to use a `-pytorch` suffix.

Reply:
This is more complicated, than you think. There are two model weights in that repo - pth and onnx. And yes, onnx is a converted pth weights. But we also could add another model, for some reason. And we would get naming collision. So consider `onnx` suffix as a precaution.

Reply:
OK, I suppose that's fair.